dtls_debug.h

```diff
@@ -73,6 +73,21 @@ log_t dtls_get_log_level(void);
 /** Sets the log level to the specified value. */
 void dtls_set_log_level(log_t level);
 
+/**
+ * Logging callback handler definition.
+ *
+ * @param level One of the DTLS_LOG_* values.
+ * @param message Zero-terminated string message to log.
+ */
+typedef void (*dtls_log_handler_t) (log_t level, const char *message);
```

Comment:
I would prefer, if a comment about DTLS_DEBUG_BUF_SIZE is provided here.

Reply:
Comment added and code pushed.